changelogs/unreleased/1678-slalwani97

```diff
@@ -0,0 +1 @@
+fix(maya-exporter): fix duplicate metrics generation after pool pod restart or any error.
```

Comment:
n_: no need to mention "fix" again in the changelog as it is already present in type.

Reply:
Sure, done.